lib/json-rpc-interface.js

```diff
@@ -78,6 +78,14 @@ class JSONRPCInterface extends APIInterface {
 		router.use('/jsonrpc', this.router);
 	}
 
+	sendHeader(ctx) {
+		// Send headers
+		let contentType = (ctx.routeOptions.streamingResponse) ? 'text/plain' : 'application/json';
```

Comment:
Should this also be text/plain in the case of manualResponse?

Reply:
I believe that sendHeader is never called when manualResponse is set because manualResponse means manually sending a header.